### PR TITLE
Add toggles for Reddit and Twitter feed sources

### PR DESCRIPTION
**Pull request by hanchang**
https://app.asana.com/0/683768070860536/744567967891139

### CODE REVIEW

File: app/javascript/App/components/FilterPanel/filterComponents/ToggleReddit.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      this.props.selectedItems.feedSources.includes('reddit')
+
+    return (
+      <div className="pv2">
+        <span className="mr2">
+          <img src={RedditLogo} className="mr2 v-top" />
+          Reddit
+        </span>
+        <Switch
+          on={true}
```
Comment (cryptoB0T) [inline app/javascript/App/components/FilterPanel/filterComponents/ToggleReddit.js]: pass `filterSelected` to the switch `on` attr

Comment (cryptoB0T) [inline app/javascript/App/components/FilterPanel/filterComponents/ToggleReddit.js]: pass `filterSelected` to the switch `on` attr

**Comment by cryptoB0T** [comment]
@hanchang this work is completed but not able to test on the review app. error in heroku logs, looks like the redis add-on 

```
       KeyError: key not found: "REDIS_URL"
```

**Comment by hanchang** [comment]
Yup, we recently integrated caching so all Heroku environments will need a Redis instance moving forward; I've added this to `app.json` so all new review apps will spin up with a Redis instance attached, but existing PRs need to be backfilled with Redis, which I've done for this app.

**Pull request merged**